src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -1006,6 +1016,59 @@ bool ResumptionDataDB::SelectSubscriptionsData(
   return true;
 }
 
+bool ResumptionDataDB::SelectUserLocationData(
+    const std::string& policy_app_id,
+    const std::string& device_id,
+    smart_objects::SmartObject& saved_app) const {
+  SDL_LOG_AUTO_TRACE();
+  using namespace app_mngr;
+  using namespace smart_objects;
+  uint32_t count_item = 0;
+  if (!SelectCountFromArray(
+          count_item, kSelectCountUserLocation, policy_app_id, device_id)) {
+    return false;
+  }
+
+  saved_app[strings::user_location] = SmartObject(SmartType_Map);
+
+  if (0 == count_item) {
+    SDL_LOG_INFO("Application does not contain user_location data");
```

Comment:
```suggestion
    SDL_LOG_DEBUG("Application does not contain user_location data");
```

Reply:
fixed in bf0ce24216a70351394e14311bafcfac997606f6

Reply:
fixed in 40c8b7496a1967b7dacec511f2fd014a255fa36c